source/operation.ts

```diff
@@ -61,16 +61,7 @@ export interface GetUploadMetadataOperation {
   component_id: string;
 }
 
-export type Operation =
-  | CreateOperation
-  | QueryOperation
-  | SearchOperation
-  | UpdateOperation
-  | DeleteOperation
-  | QueryServerInformationOperation
-  | QuerySchemasOperation
-  | GetUploadMetadataOperation
-  | { action: string; [key: string]: any };
+export type Operation = { action: string; [key: string]: any };
```

Comment:
Even though they all fall back to this generic type, I think it makes some sense to keep it.

Reply:
alright, readded! thanks